server/game/effects.js

```diff
@@ -562,7 +562,7 @@ const Effects = {
                 card.owner.moveCard(card, 'out of game');
             },
             unapply: function(card, context) {
-                card.owner.putIntoPlay(card);
+                card.owner.putIntoPlay(card, 'play', { isEffectExpiration: true });
```

Comment:
It's not immediately obvious what this flag is for without looking at the implementation of putIntoPlay, is the implication that it will be forced into play? if so perhaps it should be called that

Reply:
I considered that naming, but it won't quite force it into play. It will put it into play even if:
* Barring the Gates is out restricting cards from coming into play
* Iron Bank Will Have Its Due somehow activates on the card (e.g. somehow get another copy out and Iron Bank'd it - normally couldn't bring it back into play this round)
It will NOT bring the card back into play if:
* A copy of the card has ended up in the dead pile while the other copy was out of game (e.g. Tarred Heads, pillage + Ser Gregor, pillage + Crone of Vaes Dothrak, etc).

If we're okay with calling it `force` and having it NOT work when the character is in the dead pile, then we could do that. But that seems wrong. I figure we should save that for drag-drop support (which also currently won't let you drag a dead character into play, related #1104)

I could rename it `ignorePlayRestrictions` but that could also imply that it should ignore the restriction placed on the card by the dead pile.

Reply:
I see, tricky.

Doesnt feel quite right, but in lieu of a better suggestion (which i don't have right now) it can stay as is